### PR TITLE
Lps 56234

### PR DESCRIPTION
**Pull request by mtambara**

### CODE REVIEW

File: modules/apps/journal/journal-test/test/integration/com/liferay/journal/service/JournalArticleExpirationTest.java [hunk 1]
```diff
@@ -227,7 +234,7 @@ protected JournalArticle updateArticle(JournalArticle article, int mode)
 				expirationDateCalendar.get(Calendar.DAY_OF_MONTH),
 				expirationDateCalendar.get(Calendar.YEAR),
 				expirationDateCalendar.get(Calendar.HOUR_OF_DAY),
-				expirationDateCalendar.get(Calendar.MINUTE), true, 0, 0, 0, 0,
```
Comment (shuyangzhou) [inline modules/apps/journal/journal-test/test/integration/com/liferay/journal/service/JournalArticleExpirationTest.java]: Why is the neverReview changed from true to false?

**Comment by mtambara** [comment]
That's `neverExpire`, when it's true the article will get a `null` `expirationDate` and will never expire, which seemed to defeat the purpose of the test. Also, it means I can't get the `expirationDate` after the update to simulate the expiration.

**Comment by shuyangzhou** [comment]
Ok, I miss read that parameter, it is really hard to see clearly which is which when we have this crazy long parameter list :(

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/27261

**Pull request closed without merging**